README.md

```
@@ -1,67 +1,126 @@
-| Android    | API 23+ (Marshmallow) | 
+> GIFs do not represent actual performance — clone the repo and run the Samples app for a real feel.
 
-## Previews:
+<div align="center">
 
 | iOS | Android |
-| --- | --- |
-| <img src="https://user-images.githubusercontent.com/31090457/225077296-a87ef6ea-7ef4-4ab6-a904-960bcd13654b.gif" width="250" height="550"/>| <img src="https://user-images.githubusercontent.com/31090457/225077308-994f0265-463a-4109-bca1-8abc0cf634c4.gif" width="250" height="550"/> |
-| <img src="https://github.com/FreakyAli/Maui.FreakyEffects/assets/31090457/72eb3dac-b716-47ff-be96-73341ca2b1ac" width="250" height="550"/>| <img src="https://github.com/FreakyAli/Maui.FreakyEffects/assets/31090457/691abb5b-42cf-4619-9c0d-91f76d814d16" width="250" height="550"/> |
+| :---: | :---: |
+| <img src="https://user-images.githubusercontent.com/31090457/225077296-a87ef6ea-7ef4-4ab6-a904-960bcd13654b.gif" width="220"/> | <img src="https://user-images.githubusercontent.com/31090457/225077308-994f0265-463a-4109-bca1-8abc0cf634c4.gif" width="220"/> |
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)